common/changes/@rushstack/operation-graph/sennyeya-heft-lifecycle_2025-06-11-16-53.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/operation-graph",
+      "comment": "The OperationExecutionManager `beforeExecute` and `afterExecute` hooks have been made async and renamed to `beforeExecuteAsync` and `afterExecuteAsync`.",
```

Comment:
Update to include the addition of the metadata field. Also add the `(BREAKING CHANGE)` modifier because it is one.